seahorn/lib/klee_byte_buf_helper.c

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ */
+
+#include <limits.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <aws/common/common.h>
+#include <seahorn/seahorn.h>
+
+#include <bounds.h>
+#include <byte_buf_helper.h>
+#include <nondet.h>
+#include <proof_allocators.h>
+
+void initialize_byte_buf(struct aws_byte_buf *const buf) {
+    size_t cap = nd_size_t();
+    assume(cap <= sea_max_buffer_size());
+    buf->buffer = can_fail_malloc(cap * sizeof(*(buf->buffer)));
```

Comment:
this only makes sense if `sizeof(...)` is `1`. Otherwise capacity is greater than `max_buffer_size`.
But in this case, the multiplication can be dropped

Reply:
This what I follow `byte_buf_helper.c`. The source is [here](https://github.com/seahorn/verify-c-common/blob/master/seahorn/lib/byte_buf_helper.c#L25).